distribution/src/main/resources/docker-compose/nginx.conf

```
@@ -0,0 +1,86 @@
+# DISCLAIMER:
+
+# This internal config is for Dev/Test only (ie. it must *not* be used for live / production environments with private /confidential data)
+# It is unsupport and for internal reference only It is insecure & not tuned for performance. Test data will be lost (ie. no upgrade testing)
+
+# note: originally adapted from: https://github.com/Alfresco/acs-ingress/blob/master/nginx.conf
+
+
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    server {
+        listen *:8080;
+
+        client_max_body_size 0;
+
+        set  $allowOriginSite *;
+
+        proxy_pass_request_headers on;
+
+        proxy_pass_header Set-Cookie;
+
+        # External settings, do not remove
+        #ENV_ACCESS_LOG
+
+        proxy_next_upstream error timeout invalid_header http_500 http_502 http_503 http_504;
+        proxy_redirect off;
+        proxy_buffering off;
+
+        # Added for Keycloak (/auth) to workaround "upstream sent too big header ..." error. For Dev/Test only (not "tuned")
+        proxy_buffer_size          12k;
+
+        proxy_set_header Host            $host:$server_port;
+        proxy_set_header X-Real-IP       $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_pass_header Set-Cookie;
+
+        # Protect access to SOLR APIs
+        location ~ ^(/.*/service/api/solr/.*)$ {return 403;}
+        location ~ ^(/.*/s/api/solr/.*)$ {return 403;}
+        location ~ ^(/.*/wcservice/api/solr/.*)$ {return 403;}
+        location ~ ^(/.*/wcs/api/solr/.*)$ {return 403;}
+
+        location ~ ^(/.*/proxy/alfresco/api/solr/.*)$ {return 403 ;}
+        location ~ ^(/.*/-default-/proxy/alfresco/api/.*)$ {return 403;}
+
+        # Protect access to Prometheus endpoint
+        location ~ ^(/.*/s/prometheus)$ {return 403;}
+
+        location / {
+            proxy_pass http://alfresco:8080;
+        }
+
+        location /workspace/ {
+            proxy_pass http://digital-workspace:8080/;
+
+            # If using external proxy / load balancer (for initial redirect if no trailing slash)
+            absolute_redirect off;
+        }
+
+        location /alfresco/ {
+            proxy_pass http://alfresco:8080;
+
+            # If using external proxy / load balancer (for initial redirect if no trailing slash)
+            absolute_redirect off;
+        }
+
+        location /auth/ {
```

```suggestion
        location /auth {
```

@mpichura try with this

trailing slash in location directive usually is not a good thing to have, depending on how you configure proxy_pass (with a trailing slash or not) it may chop or add an additional (double) slash in the final request being forwarded upstream

---

Thanks @gionn for pointing this out - I've removed all trailing slashes